doc/visual-programming/source/widgets/data/sql.rst

```diff
@@ -0,0 +1,27 @@
+
+SQL Table
+=========
+
+.. figure:: icons/file.png
```

Comment:
This uses the wrong icon. Maybe @ajdapretnar can help with getting the right one from the widget to doc (svg->png?).